src/dotnet/CarbonAware.WebApi/Controllers/CarbonAwareController.cs

```diff
@@ -58,10 +60,11 @@ public async Task<IActionResult> GetEmissionsDataForLocationsByTime([FromQuery(N
     [ProducesResponseType(StatusCodes.Status400BadRequest)]
     [HttpGet("bylocation")]
     public async Task<IActionResult> GetEmissionsDataForLocationByTime(string location, DateTime? time = null, DateTime? toTime = null, int durationMinutes = 0)
-    {
+    {;
+        var loc = new Location(){ RegionName = location };
```

Comment:
```suggestion
        var locations =  new List<Location>(){ new Location(){ RegionName = location } };
```

and correspondingly below